From: Alexander Mikhaylenko <alexander.mikhaylenko@puri.sm>
Date: Wed, 23 Sep 2020 18:48:50 +0500
Subject: librem5: bookmarks-row: Make properties dialog resizable

Ensure it properly maximizes and keyboard doesn't overlap it.

Upstreamed in https://gitlab.gnome.org/GNOME/epiphany/-/merge_requests/832
---
 src/bookmarks/ephy-bookmark-row.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/bookmarks/ephy-bookmark-row.c b/src/bookmarks/ephy-bookmark-row.c
index e060931..42f845a 100644
--- a/src/bookmarks/ephy-bookmark-row.c
+++ b/src/bookmarks/ephy-bookmark-row.c
@@ -62,7 +62,6 @@ ephy_bookmark_row_button_clicked_cb (EphyBookmarkRow *row,
                          "title", _("Bookmark Properties"),
                          "transient-for", GTK_WINDOW (gtk_widget_get_toplevel (GTK_WIDGET (row))),
                          "use-header-bar", TRUE,
-                         "resizable", FALSE,
                          "modal", TRUE,
                          NULL);
 
